<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenshinImpact_AutoMapClass</class>
 <widget class="QMainWindow" name="GenshinImpact_AutoMapClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>原神天理系统之自动地图</string>
  </property>
  <property name="windowIcon">
   <iconset theme="MainIcon" resource="GenshinImpact_AutoMap.qrc">
    <normaloff>:/IconMain/resource/icon.ico</normaloff>:/IconMain/resource/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="MainView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(128,128,128,0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="GenshinImpact_AutoMap.qrc">:/Initing/resource/UI/Initing.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QtClassMyExitButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>268</x>
      <y>0</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="GenshinImpact_AutoMap.qrc">
      <normaloff>:/IconExit/resource/icon32.ico</normaloff>:/IconExit/resource/icon32.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="UIButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>260</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="GenshinImpact_AutoMap.qrc">
      <normaloff>:/IconUI/resource/UI.ico</normaloff>:/IconUI/resource/UI.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="UIObjList0Button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>188</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="GenshinImpact_AutoMap_Icon.qrc">
      <normaloff>:/IconUIObjList/resource/UIObjList0.ico</normaloff>:/IconUIObjList/resource/UIObjList0.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="UIObjList1Button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>156</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="GenshinImpact_AutoMap_Icon.qrc">
      <normaloff>:/IconUIObjList/resource/UIObjList1.ico</normaloff>:/IconUIObjList/resource/UIObjList1.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="UIObjList2Button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>124</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="GenshinImpact_AutoMap_Icon.qrc">
      <normaloff>:/IconUIObjList/resource/UIObjList2.ico</normaloff>:/IconUIObjList/resource/UIObjList2.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="UIObjList101Button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="GenshinImpact_AutoMap_Icon.qrc">
      <normaloff>:/IconUIObjList/resource/UIObjList101.ico</normaloff>:/IconUIObjList/resource/UIObjList101.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="UIObjFlagButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="GenshinImpact_AutoMap_Icon.qrc">
      <normaloff>:/IconUIObjList/resource/UIObjFlag.png</normaloff>:/IconUIObjList/resource/UIObjFlag.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="UIDLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>275</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>UID: 000000000</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="UIObjList104Button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>260</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="GenshinImpact_AutoMap_Icon.qrc">
      <normaloff>:/IconUIObjList/resource/UIObjList104.ico</normaloff>:/IconUIObjList/resource/UIObjList104.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="UIObjList102Button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>260</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="GenshinImpact_AutoMap_Icon.qrc">
      <normaloff>:/IconUIObjList/resource/UIObjList102.ico</normaloff>:/IconUIObjList/resource/UIObjList102.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="UIObjList103Button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>260</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="GenshinImpact_AutoMap_Icon.qrc">
      <normaloff>:/IconUIObjList/resource/UIObjList103.ico</normaloff>:/IconUIObjList/resource/UIObjList103.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="UIObjList93Button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="GenshinImpact_AutoMap_Icon.qrc">
      <normaloff>:/IconUIObjList/resource/UIObjList93.ico</normaloff>:/IconUIObjList/resource/UIObjList93.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="UIObjList94Button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>188</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="GenshinImpact_AutoMap_Icon.qrc">
      <normaloff>:/IconUIObjList/resource/UIObjList94.ico</normaloff>:/IconUIObjList/resource/UIObjList94.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="UIObjList105Button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>220</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="GenshinImpact_AutoMap_Icon.qrc">
      <normaloff>:/IconUIObjList/resource/UIObjList105.ico</normaloff>:/IconUIObjList/resource/UIObjList105.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="UIObjList205Button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>220</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="GenshinImpact_AutoMap_Icon.qrc">
      <normaloff>:/IconUIObjList/resource/UIObjList205.ico</normaloff>:/IconUIObjList/resource/UIObjList205.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtClassMyExitButton</class>
   <extends>QPushButton</extends>
   <header>QtClassMyExitButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="GenshinImpact_AutoMap.qrc"/>
  <include location="GenshinImpact_AutoMap_UI.qrc"/>
  <include location="GenshinImpact_AutoMap_Items.qrc"/>
  <include location="GenshinImpact_AutoMap_Icon.qrc"/>
 </resources>
 <connections/>
</ui>
